apps/bundle-size/src/index.js

```diff
@@ -0,0 +1,12 @@
+// @ts-check
```

Comment:
**chore:** those inline pragmas are not needed as this rule is set within root config of the package already

Reply:
Removed in 2180054